src/CC0-1.0.xml

```diff
@@ -147,6 +147,7 @@
             </list>
         </item>
       </list>
+      <alt match="For more information, please see &lt;http://creativecommons.org/publicdomain/zero/1.0/>" name="upstreamLink"></alt>
```

Comment:
I think you need `>` -> `&gt;` in the `match` value too.

Reply:
It's not [necessary](https://www.w3.org/TR/xml/#syntax), and the validate-schema test passes. Happy to escape it though if that's stylistically preferred. The remaining failing test isn't contingent on the `>` being escaped or not (tried locally).